src/react_native/core.cljs

```diff
@@ -20,7 +21,7 @@
 (defn image-get-size [uri callback] (.getSize ^js (.-Image ^js react-native) uri callback))
 (def text (reagent/adapt-react-class (.-Text ^js react-native)))
 (def text-input (reagent/adapt-react-class (.-TextInput ^js react-native)))
-
+(def draggable-flatlist (reagent/adapt-react-class DraggableFlatList))
```

Comment:
@flexsurfer should we add this here or elsewhere?

Reply:
Removed, no need anymore